<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterMathDlg</class>
 <widget class="QDialog" name="RasterMathDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raster Math</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mpDataGroupBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTabWidget" name="mpDataTabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="mpRasterTab">
         <attribute name="title">
          <string>Rasters</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpRaster1Label">
              <property name="text">
               <string>r1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpRaster1Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpRaster2Label">
              <property name="text">
               <string>r2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpRaster2Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpRaster3Label">
              <property name="text">
               <string>r3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpRaster3Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpRaster4Label">
              <property name="text">
               <string>r4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpRaster4Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpRaster5Label">
              <property name="text">
               <string>r5:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpRaster5Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mpAoiTab">
         <attribute name="title">
          <string>AOI's</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpAoi1Label">
              <property name="text">
               <string>a1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpAoi1Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpAoi2Label">
              <property name="text">
               <string>a2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpAoi2Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpAoi3Label">
              <property name="text">
               <string>a3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpAoi3Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpAoi4Label">
              <property name="text">
               <string>a4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpAoi4Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="mpAoi5Label">
              <property name="text">
               <string>a5:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mpAoi5Combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mpFormulaGroupBox">
     <property name="title">
      <string>Formula</string>
     </property>
     <layout class="QGridLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextEdit" name="mpFormulaTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="mpFavoritesLabel">
          <property name="text">
           <string>Favorites:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mpFavoritesCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mpAddToFavoritesButton">
          <property name="toolTip">
           <string>Add to favorites...</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mpDeleteFavoriteButton">
          <property name="text">
           <string> - </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <widget class="QGroupBox" name="mpUnitsGroupBox">
       <property name="title">
        <string>Units</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mpRadiansButton">
          <property name="text">
           <string>Radians</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mpDegreesButton">
          <property name="text">
           <string>Degrees</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="mpErrorHandlingGroupBox">
       <property name="title">
        <string>Error Handling</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="topMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mpErrorFailButton">
          <property name="text">
           <string>Fail</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QRadioButton" name="mpErrorUseButton">
            <property name="text">
             <string>Use: </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mpErrorUseTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mpOutputGroupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="mpGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="mpResultNameLabel">
          <property name="text">
           <string>Result Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mpResultNameTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="mpDisplayAsLabel">
          <property name="text">
           <string>Display As:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="mpDisplayAsCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Raster Overlay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Threshold Overlay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Raster in new Window</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="mpPrecisionLabel">
          <property name="text">
           <string>Precision:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="mpPrecisionCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>6</number>
          </property>
          <item>
           <property name="text">
            <string>1-byte signed integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-byte unsigned integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-byte signed integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-byte unsigned integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-byte signed integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-byte unsigned integer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Single precision float</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double precision float</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="mpLocationLabel">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="mpLocationCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>On-Disk</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>In-memory</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Automatic</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mpButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>RasterMathDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>RasterMathDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
